opentelemetry-collector/manifest.yaml

```diff
@@ -2,61 +2,66 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-releases/core
   name: opentelemetry-collector
   description: OpenTelemetry Collector
-  version: 0.33.1
+  version: 0.34.0
   output_path: ./_build
-  otelcol_version: 0.33.1-0.20210820002854-d3000232f8f6
+  otelcol_version: 0.34.0
 
 receivers:
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver 150692dbbceb3ff0df75c912e835f1feaac0be93
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver 150692dbbceb3ff0df75c912e835f1feaac0be93
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver 150692dbbceb3ff0df75c912e835f1feaac0be93
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver 150692dbbceb3ff0df75c912e835f1feaac0be93
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver 150692dbbceb3ff0df75c912e835f1feaac0be93
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver 150692dbbceb3ff0df75c912e835f1feaac0be93
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver 9b0e90027913f53d4231eb1762063cf67df61051
```

Comment:
We should be able to replace these commit IDs with `v0.34.0` once it is tagged.

Reply:
I will do a different PR.